src/index.html

```diff
@@ -13,6 +13,76 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
```

Comment:
Fix code style everywhere
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```

Reply:
![Снимок экрана 2024-09-24 131532](https://github.com/user-attachments/assets/36ce1147-8125-4932-994d-0484c81304b2)